code/obj/item/device/transfer_valve.dm

```diff
@@ -509,17 +509,39 @@ TYPEINFO(/obj/item/device/transfer_valve/briefcase)
 /obj/item/pressure_crystal
 	icon = 'icons/obj/items/assemblies.dmi'
 	icon_state = "pressure_3"
-	var/pressure = 0
-	var/total_pressure = 0
-	desc = "A pressure crystal. We're not really sure how it works, but it does. Place this near where the epicenter of a bomb would be, then detonate the bomb. Afterwards, place the crystal in a tester to determine the strength."
-	name = "Pressure Crystal"
+	w_class = W_CLASS_SMALL
+	var/pressure = 0 // used to calculate credit value, in shippingmarket.dm proc/appraise_value
+	var/last_explode_time = 0
+	var/static/explosion_id = 0
+	desc = "A pressure crystal. We're not really sure how it works, but it does. Place this near where the epicenter of a bomb would be, \
+		then detonate the bomb. Afterwards, place the crystal in a tester to determine the strength.<br>\
+		Spent pressure crystals can be sold to researchers on the shipping market, for a credit sum depending on the measured power."
```

Comment:
This seems... really verbose.  Should some of this be moved over to a help message?

Reply:
Ahh forgot about those, good point. It is done